modules/ztp-factory-provisioning-hub-prereqs.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="factory-installing-edge-sites_{context}"]
+= Hub node prerequisites
```

Comment:
I don't see this in the preview, should it be included somewhere, or removed?

Reply:
I need to remove

Reply:
Can you please remove this file?